src/ReadLine/ReadLine.csproj

```diff
@@ -3,32 +3,57 @@
   <PropertyGroup>
```

Comment:
I updated this file on master, you should do a rebase

Reply:
Not showing that I need to rebase.   I pulled everything but there was nothing to pull / merge that conflicted with mine which 'was' technically correct.   Now it seems to not sync with compiling after submission.